geode-core/src/upgradeTest/java/org/apache/geode/security/AuthExpirationMultiServerDUnitTest.java

```diff
@@ -253,6 +254,53 @@ public void registerInterestsWithMultiServers() throws Exception {
         .containsExactly("DATA:READ:partitionRegion:key0");
   }
 
+  @Test
+  public void consecutivePut() throws Exception {
+    int locatorPort = locator.getPort();
+    // do consecutive puts using a client
+    ClientVM client = cluster.startClientVM(3,
+        c -> c.withProperty(SECURITY_CLIENT_AUTH_INIT, UpdatableUserAuthInitialize.class.getName())
+            .withCacheSetup(ccf -> ccf.setPoolMaxConnections(2))
+            .withLocatorConnection(locatorPort));
+    AsyncInvocation invokePut = client.invokeAsync(() -> {
+      UpdatableUserAuthInitialize.setUser("user1");
+      Region<Object, Object> proxyRegion =
+          ClusterStartupRule.getClientCache()
```

Comment:
`Objects.requireNonNull(ClusterStartupRule.getClientCache())` reduces warnings

Reply:
Just my personal preference: I feel like I rather live with the warning than clutter up the code.

Reply:
Just my 2 cents, but this warning shows up all over the place, so I wonder if it might be better to just make `ClusterStartupRule.getCache()` and `ClusterStartupRule.getClientCache()` throw an NPE if they're called with a null cache, since that's what would happen anyway.